### PR TITLE
Add GetBestLwsFromGws in PermuteKernel_tile_8x8

### PR DESCRIPTION
**Pull request by EunsooSheenIntel**

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp [hunk 1]
```diff
@@ -42,6 +43,52 @@ ParamsKey PermuteKernel_tile_8x8::GetSupportedKey() const {
     return k;
 }
 
+std::vector<size_t> PermuteKernel_tile_8x8::GetBestLwsFromGws(const std::vector<size_t>& gws, const size_t tile_height, const size_t tile_width)
+{
+    std::vector<size_t> lws{1,1,1};
+
+    // float32 * tile_height * tile_width * 2 * work_items <= 64K
+    size_t max_num_work_items = 65536 / (4 * tile_height * tile_width * 2);
```
Comment (yeonbok) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp:51]: I updated latest code to use one buf. So we dont need to use x2 any more. 
So max num work items will be min(256, (64*1024)/(sizeof(input type) * hight* width) )

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp [hunk 2]
```diff
@@ -17,7 +17,10 @@
 #include "kernel_selector_utils.h"
 #include <string>
 #include <functional>
+#include <cmath>
+
 #define CEIL_DIV(A, B) ((A + B - 1)/(B))
+#define VECTORWIDTH 8
```
Comment (yeonbok) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp:23]: please remove this, this is currently define din param.h

Comment (yeonbok) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp:23]: (of course that will be moved to permute_kernel_tile_8x8.h...

Comment (EunsooSheenIntel) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp]: Okay, I removed VECTOWIDTH macro

**Pull request merged**